editors/sc-ide/CMakeLists.txt

```diff
@@ -350,21 +350,29 @@ if(APPLE)
                 #fixup_qt5_executable(\"${CMAKE_INSTALL_PREFIX}/SuperCollider/SuperCollider.app\")
                 " COMPONENT Runtime)
 
-
+elseif(CMAKE_GENERATOR MATCHES "Visual Studio")
+  set_property(TARGET SuperCollider PROPERTY RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/editors/sc-ide")
+  install(TARGETS SuperCollider
+      CONFIGURATIONS Debug;Release
+      DESTINATION "SuperCollider"
+      PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+  install(FILES ${translations}
+      CONFIGURATIONS Debug;Release
```

Comment:
> someone just needs to do it
Yea, or more precise, somebody who knows how to do it needs to do it. Using a dll would force us to provide a binary, as portaudio is not distributed as binary, or expect people to build their own dll. Building a portaudio dll is even more fragile than a static library, I haven't digged deep into @sonoro1234 cmake mods, but I did see errors when building more than just the static library. (Check Google, I am not the only one with this problem. The build configuration pulished here (https://github.com/slowriot/portaudio-mingw-w64) breaks when ASIO support is added).
My conclusion tends to be, embed portaudio and have a static library being built. Can you or somebody else give pointers of how to do that?